team-service-client-spring/src/main/java/org/zalando/stups/fullstop/teams/RestTemplateTeamOperations.java

```diff
@@ -45,10 +45,15 @@ public RestTemplateTeamOperations(final RestOperations restOperations, final Str
     public List<Account> getAwsAccountsByUser(final String userId) {
         Preconditions.checkArgument(StringUtils.hasText(userId), "userId must not be blank");
 
-        final ResponseEntity<List<Account>> response = restOperations.exchange(
+        final ResponseEntity<List<Account>> typeAws = restOperations.exchange(
                 get(URI.create(baseUrl + "/api/accounts/aws?member=" + userId)).build(), userTeamListType);
-        Preconditions.checkState(response.getStatusCode().is2xxSuccessful(), "getAwsAccountsByUser failed: %s", response);
-        return response.getBody();
+        Preconditions.checkState(typeAws.getStatusCode().is2xxSuccessful(), "getAwsAccountsByUser for type AWS failed: %s", typeAws);
+        final ResponseEntity<List<Account>> typeK8s = restOperations.exchange(
+                get(URI.create(baseUrl + "/api/accounts/kubernetes?role=PowerUser&member=" + userId)).build(), userTeamListType);
+        Preconditions.checkState(typeAws.getStatusCode().is2xxSuccessful(), "getAwsAccountsByUser for type Kubernetes failed: %s", typeK8s);
+        final List<Account> allAccounts = typeAws.getBody();
```

Comment:
not sure if the list returned from `.getBody()` is mutable. Better create a completely new list and the add the members from both.